Broken TOKEN

```
===== pyelftools-0.23-py_0 OK =====
TEST END: pyelftools-0.23-py_0
# If you want to upload this package to anaconda.org later, type:
#
# $ anaconda upload /feedstock_root/build_artefacts/noarch/pyelftools-0.23-py_0.tar.bz2
#
# To have conda build upload to anaconda.org automatically, use
# $ conda config --set anaconda_upload yes

Using Anaconda Cloud api site https://api.anaconda.org
Using Anaconda Cloud api site https://api.anaconda.org
[ERROR] 
Traceback (most recent call last):
  File "/opt/conda/bin/anaconda", line 6, in <module>
    sys.exit(binstar_client.scripts.cli.main())
  File "/opt/conda/lib/python3.5/site-packages/binstar_client/scripts/cli.py", line 87, in main
    description=__doc__, version=version)
  File "/opt/conda/lib/python3.5/site-packages/binstar_client/scripts/cli.py", line 66, in binstar_main
    return args.main(args)
  File "/opt/conda/lib/python3.5/site-packages/binstar_client/commands/upload.py", line 173, in main
    raise errors.BinstarError('file %s does not exist' % (filename))
binstar_client.errors.BinstarError: file /feedstock_root/build_artefacts/linux-64/pyelftools-0.23-py_0.tar.bz2 does not exist
Traceback (most recent call last):
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 124, in <module>
    main()
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 108, in main
    upload(cli, meta, owner, channel)
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 55, in upload
    env=os.environ)
  File "/opt/conda/lib/python3.5/subprocess.py", line 584, in check_call
    raise CalledProcessError(retcode, cmd)

subprocess.CalledProcessError: Command '['anaconda', '--quiet', '-t', 'binstar.token', 'upload', '/feedstock_root/build_artefacts/linux-64/pyelftools-0.23-py_0.tar.bz2', '--user=conda-forge', '--channel=main']' returned non-zero exit status 1 ./ci_support/run_docker_build.sh returned exit code 1
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -8,7 +8,6 @@ source:
   md5: aa7cefa8bd2f63d7b017440c9084f310
 
 build:
-  noarch_python: True
   skip: True  # [not linux]
 
 requirements:

